starter-code/javascript/chronometer.js

```diff
@@ -1,39 +1,78 @@
 // Constructor
-// function Chronometer() {
+function Chronometer() {
+  this.currentTime = 0;
+  this.intervalId;
+}
```

Comment:
Si inicializais la propiedad intervalId...
this.intervalId = 0;
... os pasa el jasmine y además yo creo que es mejor, os puede ahorrar problemas.